changelog/fix-6941-warning-for-negative-balance

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
+Comment: Fixed a bug where the 'deposits paused while balance is negative' notice was erroneously shown after an instant deposit.
```

Comment:
❓ Is this intentionally a "placeholder" changelog entry, or should it be a user-facing changelog entry.

Reply:
TIL regarding user-facing changelogs. I always just use best judgement. Updated in https://github.com/Automattic/woocommerce-payments/pull/8117/commits/24975379bc7a6fa1752def09b12e02c1c467986c, thanks.

Reply:
User-facing is also just how I approach it, unsure if there are specific guidelines for this 🙈